core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -225,25 +227,44 @@ public AtomicReference<Long> getWorkerId() {
   }
 
   @Override
-  public FileInfo getFileInfo(String fileId) throws IOException {
+  public FileInfo getFileInfo(String ufsFullPath, GetStatusPOptions options) throws IOException {
     alluxio.grpc.FileInfo fi;
-    String ufsFullPath = fileId; // File ID is full ufs path passed from client
-    String fn = new AlluxioURI(fileId).getName();
+    long syncIntervalMs = options.hasCommonOptions()
+        ? options.getCommonOptions().getSyncIntervalMs() :
```

Comment:
Do we need to check 
options.getCommonOptions().hasSyncIntervalMs()? or what's the default value for options.commonoptions.syncIntervalMs?

Reply:
We always initialize the "syncIntervalMs" field when we prepare the "commonOptions" for GetStatusPRequest in current code.
But you are right. We need to check existence of every "optional" field in the message before accessing it.

Fixed as you suggested.